cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/stream/FileStreamAdmin.java

```diff
@@ -296,6 +297,13 @@ public StreamConfig call() throws Exception {
     });
   }
 
+  private void assertNamespaceHomeExists(Id.Namespace namespaceId) throws IOException {
+    Location namespaceHomeLocation = Locations.getParent(getStreamsHomeLocation(namespaceId));
+    Preconditions.checkArgument(namespaceHomeLocation.exists(),
+                                String.format("Expected home directory %s for namespace %s to exist.",
```

Comment:
Should we standardize this message between the app deploy, dataset module deploy and this methods? They essentially do the same thing, even if they are defined separately, it may make sense to at lease define the same message for all three.